docs/technical_reference/flowsheets/ion_exchange.rst

```diff
@@ -0,0 +1,124 @@
+    * Finally, the model is solved for these new conditions.
+    
+    Note that stopping at this point will likely result in a non-integer value for ``number_columns``. This value also corresponds to an optimal value for both ``bed_depth`` and 
+    ``dimensionless_time``. The user can optionally ensure the ``number_columns`` is an integer by rounding the ``number_columns`` to an integer value after 
+    the initial optimization solve, fixing ``number_columns`` to that value, and then re-solving the model to obtain the final values for ``bed_depth`` and ``dimensionless_time``.
+    This is the approach taken in the demonstration file.
+        
+
+The following modeling components are used within the flowsheet:
+
+Documentation for property models:
+    * :doc:`/technical_reference/property_models/mc_aq_sol`
+Documentation for unit models:
+    * :doc:`/technical_reference/unit_models/ion_exchange_0D`
+Documentation for unit models from IDAES:
+    * :doc:`idaes:reference_guides/model_libraries/generic/unit_models/feed`
+    * :doc:`idaes:reference_guides/model_libraries/generic/unit_models/product`
+Documentation for costing models:
+    * :doc:`/technical_reference/costing/watertap_costing`
+    * :doc:`/technical_reference/costing/ion_exchange`
+
+Degrees of Freedom
+------------------
+
+The degrees of freedom for the flowsheet can change depending on the configuration options specified during the build. 
+Specifics for other configurations options are available in the IX model documentation.
+In this demonstration, the following variables are initially fixed for simulating the IX flowsheet (i.e., degrees of freedom = 0):
+
+    * feed conditions (component flows, temperature, pressure)
+    * Langmuir equilibrium coefficient for target ion
+    * Resin capacity, diameter, porosity, and density
+    * Number of columns for the system
+    * Service flow rate
+    * The dimensionless time for the constant-pattern solution
+
+Flowsheet Specifications
+------------------------
+
+.. csv-table::
+   :header: "Description", "Value", "Units"
+
+   "Feed molar flowrate of water", "2777.5", ":math:`\text{mol}/\text{s}`"
+   "Feed molar flowrate of target ion", "0.125", ":math:`\text{mol}/\text{s}`"
+   "Feed temperature", "298.15", ":math:`\text{K}`"
+   "Feed pressure", "101325", ":math:`\text{Pa}`"
+   "Langmuir equilibrium coefficient", "0.7", ":math:`\text{dimensionless}`"
+   "Resin capacity", "3.0", ":math:`\text{mol/kg}`"
+   "Resin bulk density", "0.7", ":math:`\text{kg/L}`"
+   "Constant pattern dimensionless time", "1", ":math:`\text{dimensionless}`"
+   "Number columns", "4", ":math:`\text{dimensionless}`"
+   "Bed void fraction", "0.5", ":math:`\text{dimensionless}`"
+   "Bed depth", "1.7", ":math:`\text{m}`"
+   "Service flow rate", "15", ":math:`\text{BV/hr}`"
```

Comment:
```suggestion
   "Service flow rate", "15", ":math:`\text{V_{bed}/hr}`"
```
Similar to the above note, I was also not familiar with what BV meant before looking it up. So perhaps replacing it with this will make it more clear to readers less familiar with IX terminology

Reply:
hmmm yeah that is tricky. BV is a dimensionless term and is meant to represent the _number_ of bed volumes. But I see your point. What you have here implies the units would be m3/hr, but it is actually 1/hr.